cfgov/v1/atomic_elements/organisms.py

```diff
@@ -860,18 +874,35 @@ class Meta:
     class Media:
         js = ['filterable-list.js']
 
+    @staticmethod
+    def get_filterable_topics(filterable_page_ids, value):
+        """Given a set of page IDs, return the list of filterable topics"""
+        tags = Tag.objects.filter(
+            v1_cfgovtaggedpages_items__content_object__id__in=filterable_page_ids  # noqa E501
+        ).values_list('slug', 'name')
+
+        sort_order = value.get('topic_filtering', 'sort_by_frequency')
+        if sort_order == 'sort_alphabetically':
+            return tags.distinct().order_by('name')
+        elif sort_order == 'sort_by_frequency':
+            return [tag for (tag, _) in Counter(tags).most_common()]
+        else:
+            return []
+
     def get_context(self, value, parent_context=None):
         context = super(FilterableList, self).get_context(
             value,
             parent_context=parent_context
         )
-
+        show_topics = value['topic_filtering'] == 'sort_by_frequency' or \
+            value['topic_filtering'] == 'sort_alphabetically',
```

Comment:
There is an extra trailing comma at the end of this line that turns this into a tuple. This means that even if the value is false, it actually gets set to `(False,)` which is truthy. This causes a bug where an empty topic list always gets rendered even if you select "Don't filter topics":

<img width="281" alt="image" src="https://user-images.githubusercontent.com/654645/64869136-bf5dea00-d60e-11e9-8411-0ec574389486.png">

Reply:
How embarrassing. I've corrected the typo and tested again locally. I believe this is fixed.